applications/emocam/emocam1.js

```diff
@@ -33,7 +33,9 @@ var webWindow;
     var yaw = 0;
     var roll = 0;
     var headTransform = null;
-    var lastDataArrived = Date.now();
+    var lastDataArrived = Date.now() - 3000;
```

Comment:
Why -3000? And why -anything?

Surely the arrival time for anything is "now" and not in the past.

Reply:
This is because otherwise application thinks that data has arrived upon starting, and avatar's head is twisted for the first 2 seconds due to this. It is indeed unclear currently, so maybe we should set it to 0 instead?

Reply:
I see, that makes sense! Yeah, I think 0 here would be much clearer

Reply:
Fixed :)